software/gui/src/qserial_output_stream.h

```diff
@@ -25,11 +28,11 @@ class QSerialOutputStream : public OutputStream {
 
   StreamResponse put(int32_t b) override {
     if (EndOfStream == b) {
-      return {.count_written_ = 0, .flags = ResponseFlags::StreamSuccess};
+      return {0, ResponseFlags::StreamSuccess};
     }
     char byte = static_cast<char>(b);
     port_->write(&byte, 1);
-    return {.count_written_ = 1, .flags = ResponseFlags::StreamSuccess};
+    return {1, ResponseFlags::StreamSuccess};
```

Comment:
Could we keep some detail in comments? As in:
```suggestion
return {/*.count_written=*/1, ResponseFlags::StreamSuccess};
```
(same for all those similar ones)